pnpm/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "pnpm",
+  "version": "0.0.3",
+  "description": "PNPM",
+  "main": "index.js",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FormidableLabs/monorepo-infra-experiments.git"
+  },
+  "author": "Ryan Roemer <ryan.roemer@formidable.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/monorepo-infra-experiments/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/monorepo-infra-experiments#readme",
+  "scripts": {
+    "build": "pnpm -r --filter='./packages/**' run build"
```

Comment:
Locally, I was able to just run `pnpm -run -r build` and it worked the same as this; I'm assuming that it's picking up the `pnpm-workspace.yaml` config?

Reply:
Yeah, I think it picks it up the same way.